color_helper.py

```diff
@@ -72,6 +72,9 @@ def start_file_index(view):
                     if s.get('show_index_status', True):
                         sublime.status_message('File color indexer started...')
 
+def preview_is_on_left():
+    """Return boolean for positioning preview on left/right"""
```

Comment:
I know, I actually made the comment before you fixed it but finished the review yesterday.